corehq/apps/reports/standard/sms.py

```diff
@@ -319,16 +349,22 @@ def fields(self):
 
     @property
     def headers(self):
-        header = DataTablesHeader(
+        headers = DataTablesHeader(*[header for header in [
             DataTablesColumn(_("Timestamp")),
             DataTablesColumn(_("User Name")),
             DataTablesColumn(_("Phone Number")),
             DataTablesColumn(_("Direction")),
             DataTablesColumn(_("Message")),
+            DataTablesColumn(_("Status")) if self.testing_changes else Ellipsis,
+            DataTablesColumn(_("Event"), sortable=False) if self.testing_changes else Ellipsis,
```

Comment:
Why `Ellipsis` instead of `None`?

Reply:
I did return `None` initially (and used `filter(None, [])`), but then when I do this same pattern below, I was worried about other falsy types like the empty string being inadvertently excluded, so I figured `Ellipsis` would be more explicit.  Then I went back and changed this one for parity.

Reply:
`[x for x in ... if x is not None]` is better than `filter(None, ...)`